src/app/components/BackButton/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { Image, TouchableOpacity } from 'react-native';
+import { withNavigation } from 'react-navigation';
+
+import styles from './styles';
+import backImage from '../../assets/ic_back.png';
+
+class BackButton extends Component {
```

Comment:
maybe a stateless componente will be better

Reply:
Even better will be to extract the onPress function in an inside function of the component